packages/hash/design-system/tsconfig.json

```
@@ -0,0 +1,9 @@
+{
+  "extends": "../../../tsconfig.base.json",
+  "compilerOptions": {
+    "jsx": "preserve",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "noEmit": true
+  },
+  "include": ["assets.d.ts", "src/theme-override.d.ts", "**/*.ts", "**/*.tsx"]
```

```suggestion
  "include": ["assets.d.ts", "src/theme-override.d.ts", "src/**/*.ts", "src/**/*.tsx"]
```

Not 100% sure about this, but might be worth checking this against other packages / TS docs.

---

Thanks! Checked and updated in [fa20f71](https://github.com/hashintel/hash/pull/601/commits/fa20f7156154c4498e66c935214cb4c8c65837f1). Also realized I don't need to do the hack where I import theme-overrides.d.ts with this change